### DISCUSSION TITLE
Question about Amazon

### DISCUSSION THREAD

**Discussion by the-skiddo** [QuiteAFancyEmerald/InvisiProxy#69] [Q&A]
Hey! What do you suggest in order to get onto Amazon? Thanks! 

_**also funny number**_

**Comment by QuiteAFancyEmerald** [comment]
I don't really know what proxy would best suited for this but if you are simply trying to view the site I suppose either Alloy or Womginx would work. lol why Amazon though?

**Reply by foxsouns** [reply]
@SkidFace i personally hover around newegg for computer parts stuff. even newegg is blocked for you, corrosion runs it surprisingly well.
